lutris/runners/libretro.py

```diff
@@ -94,6 +99,7 @@ class libretro(Runner):
         'Sharp X68000',
         'Sony PlayStation Portable',
         'Sega Dreamcast',
+        'Sega Dreamcast',
```

Comment:
nope, redream is a dreamcast emu and reicast is a dreamcast emu, and red comes before rei, tried to keep to the alphabetic order that was already there in the core election order, thus here it look like a double entry when it isn't really